packages/backend/src/business/checkInvoiceState.ts

```diff
@@ -5,20 +5,20 @@ import { OFFER_STATE } from "@openPleb/common/types";
 import { eq } from "drizzle-orm";
 import { eventEmitter } from "../events/emitter";
 
-export const checkInvoiceState = async (
+export const checkInvoiceState = async ( //check the invoice state
```

Comment:
```suggestion
export const checkInvoiceState = async ( 
```
the function name already gives us the same information